ROLES.md

```diff
@@ -126,51 +91,53 @@ issues and PRs assigned to them, participate in working group meetings, and
 pre-submit tests are automatically run for their PRs. Members are expected to
 remain active contributors to the community.
 
-All members are encouraged to help with the code review burden, although each PR
-must be reviewed by an official [Approver](#approver).
+All members are encouraged to help with code reviews, although each PR
+must be approved by an official [Approver](#approver).
 
 When reviewing, members should focus on code quality and correctness, including
 testing and factoring. Members might also review for more holistic issues, but
 this is not a requirement.
 
 ### Requirements
 
-- Member of the Knative github org. Create a PR adding you to
+- Member of the Knative github org
+  - Create a PR adding yourself as a Member to
 [knative.yaml](../main/peribolos/knative.yaml) and/or to
-[knative-sandbox.yaml](../main/peribolos/knative-sandbox.yaml) which after when
-merged will send you an invite that you have to accept to become a member
-of these organizations.
+[knative-sandbox.yaml](../main/peribolos/knative-sandbox.yaml)
+  - After the PR is merged you will receive an invite that you must accept to become a Member
+  - If you would like to work towards becoming an Approver, open a second PR and add yourself as a `reviewer` on the repositories to which you will be contributing.
 
 - Has made multiple contributions to the project or community. Contributions
   might include, but are not limited to:
 
-  - Authoring and reviewing PRs on GitHub.
-
-  - Filing and commenting on issues on GitHub.
-
-  - Contributing to working group or community discussions.
+  - Authoring and reviewing PRs on GitHub
+  - Submitting and commenting on issues on GitHub
+  - Triaging and resolving issues on GitHub
+  - Contributing to working group or community discussions
 
 - Subscribed to
-  [knative-dev@googlegroups.com](https://groups.google.com/forum/#!forum/knative-dev).
+  [knative-dev@googlegroups.com](https://groups.google.com/forum/#!forum/knative-dev)
+
+- Actively contributing to 1 or more areas
 
-- Actively contributing to 1 or more areas.
+- Nominated by two Knative Members, at least one of whom does not work for the same employer
 
 ### Responsibilities and privileges
 
-- Responsive to issues and PRs assigned to them.
+- Responsive to issues and PRs assigned to them
 
 - Active owner of code they have contributed (unless ownership is explicitly
-  transferred).
+  transferred)
 
-  - Code is well tested.
+  - Code is well tested
```

Comment:
```suggestion
  - Contributions are well tested
```

Making this less code specific. For example, website contributions could be content, but tests for things like the website framework still need to be run and pass.